lib/browser/mqtt_request_response/protocol_adapter.ts

```diff
@@ -461,4 +469,17 @@ export class ProtocolClientAdapter extends BufferedEventEmitter {
                 return false;
         }
     }
+
+    private static isSubackReasonCodeRetryable(reasonCode: mqtt5.SubackReasonCode) : boolean {
+        switch (reasonCode) {
+            case mqtt5.SubackReasonCode.UnspecifiedError:
```

Comment:
Debatable: UnspecifiedError seems like something we don't want to retry. Seems like the kind of thing where a retry wouldn't resolve but I don't have strong enough feelings about it if you put it here for a reason.

Reply:
Removed